docs/releasing.md

```diff
@@ -1,6 +1,19 @@
 # Kubernetes Releases
 
-TBD
+This repo contains the tooling and documentation for the release of
+the core Kubernetes project.  
+
+In the future it is expected that the functionality will expand and be
+generalized to support release infrastrcture for all of the kubernetes
+sub-projects as well.
+
+The tooling and doc here is expected to change often as requirements
+change and the project(s) evolve.
+
+The doc and tooling in this repo is NOT designed to address the planning,
+coordination of releases.  See ---insert goltermann and PM project
```

Comment:
Best docs currently are the community wiki for timelines (https://github.com/kubernetes/community/wiki/Release-1.4) and the features repo for feature planning (https://github.com/kubernetes/features).